importer/api/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""
+Import OLX data into LORE.
+"""
+
+# stdlib
+from glob import glob
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join
+import tarfile
+import zipfile
+
+# PyPi
+
+from xbundle import XBundle, DESCRIPTOR_TAGS
+from lxml import etree
+
+# MITODL
+# commented out for now, until it's merged
+# from lore.learningobjects.models import Course, LearningObject
+
+def import_course_from_file(filename):
```

Comment:
This is assuming you are getting a file name, I don't think that is a safe assumption, I think it would probably be better to assume this is a file class so we can use something like django storage or other file abstraction, after it is unzipped, you can go back to using regular filename/path semantics